### PR TITLE
Add evictLeaderTimeout for the examples where replicas of TiKV is 1

### PR DESCRIPTION
**Pull request by DanielZhangQD**
### Code changes

- [ ] Has Go code change
- [ ] Has CI related scripts change

### Tests

- [ ] Unit test 
- [ ] E2E test 
- [ ] Manual test 
- [ ] No code 

### Side effects

- [ ] Breaking backward compatibility 
- [ ] Other side effects: 

### Related changes

- [ ] Need to cherry-pick to the release branch 
- [ ] Need to update the documentation 

### Release Notes

Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/blob/master/docs/release-note-guide.md) before writing the release note.

```release-note

```

### CODE REVIEW

File: examples/basic-cn/tidb-cluster.yaml [hunk 1]
```diff
@@ -25,6 +25,7 @@ spec:
     config: {}
   tikv:
     baseImage: uhub.service.ucloud.cn/pingcap/tikv
+    evictLeaderTimeout: 1m
```
Comment (csuzhangxc) [inline examples/basic-cn/tidb-cluster.yaml:28]: put this line nearer to the `replicas: 1 and a comment to explain it?

File: examples/basic-cn/tidb-cluster.yaml [hunk 2]
```diff
@@ -26,6 +26,10 @@ spec:
   tikv:
     baseImage: uhub.service.ucloud.cn/pingcap/tikv
     maxFailoverCount: 0
+    # If only 1 TiKV is deployed, the TiKV region leader 
+    # cannot be transferred during upgrade, so we have
+    # to configure a short timeout here
```
Comment (DanielZhangQD) [inline examples/basic-cn/tidb-cluster.yaml:31]:
```suggestion
    # to configure a short timeout
```

File: examples/basic-storagevolumes/tidb-cluster.yaml [hunk 3]
```diff
@@ -26,6 +26,10 @@ spec:
   tikv:
     baseImage: pingcap/tikv
     maxFailoverCount: 0
+    # If only 1 TiKV is deployed, the TiKV region leader 
+    # cannot be transferred during upgrade, so we have
+    # to configure a short timeout here
```
Comment (DanielZhangQD) [inline examples/basic-storagevolumes/tidb-cluster.yaml:31]:
```suggestion
    # to configure a short timeout
```

File: examples/basic-tls/tidb-cluster.yaml [hunk 4]
```diff
@@ -29,6 +29,10 @@ spec:
   tikv:
     baseImage: pingcap/tikv
     maxFailoverCount: 0
+    # If only 1 TiKV is deployed, the TiKV region leader 
+    # cannot be transferred during upgrade, so we have
+    # to configure a short timeout here
```
Comment (DanielZhangQD) [inline examples/basic-tls/tidb-cluster.yaml:34]:
```suggestion
    # to configure a short timeout
```

File: examples/basic/tidb-cluster.yaml [hunk 5]
```diff
@@ -24,8 +24,12 @@ spec:
       storage: "1Gi"
     config: {}
   tikv:
-    baseImage: pingcap/tikv
+    baseImage: pingcap/tikv
```
Comment (DanielZhangQD) [inline examples/basic/tidb-cluster.yaml:27]:
```suggestion
    baseImage: pingcap/tikv
```

File: examples/basic/tidb-cluster.yaml [hunk 6]
```diff
@@ -24,8 +24,12 @@ spec:
       storage: "1Gi"
     config: {}
   tikv:
-    baseImage: pingcap/tikv
+    baseImage: pingcap/tikv    
     maxFailoverCount: 0
+    # If only 1 TiKV is deployed, the TiKV region leader 
+    # cannot be transferred during upgrade, so we have
+    # to configure a short timeout here
```
Comment (DanielZhangQD) [inline examples/basic/tidb-cluster.yaml:31]:
```suggestion
    # to configure a short timeout
```

**Comment by DanielZhangQD** [comment]
@csuzhangxc PTAL again.

**Review by csuzhangxc — APPROVED** [review]

**Review by KanShiori — APPROVED** [review]

**Pull request merged**